ytfzf

```diff
@@ -304,23 +308,23 @@ EOF
 	fi
 
 	# selecte format if flag given
-	[ $show_format -eq 1 ] && {
+	if [ $show_format -eq 1 ]; then 
 		YTFZF_PREF="$(youtube-dl -F "$(printf "$urls" | sed 1q)" | sed '1,3d' | tac - |\
 		eval "$prompt_menu" | sed -E 's/^([^ ]*) .*/\1/')"
 		[ -z $YTFZF_PREF  ] && exit;
-	}
+	fi	
 }
 play_url () {
 	[ $YTFZF_CUR -eq 1 ] && printf "$selected_data" > "$current_file" ;
 
-	[ -n "$YTFZF_PREF" ] && {
+	if [ -n "$YTFZF_PREF" ]; then
 		eval "$player_format"\'$YTFZF_PREF\'  "\"$(printf "$urls" | tr '\n' '\t' | sed 's_\t_" "_g' )\""
-	} || {
```

Comment:
This line is needed as is
The intension here is, that if the format is not available then it will play in without prefered format. 
Leave the if statements in the play_url function. The rest is good